### PR TITLE
Create Registration component with CSS module

### PR DESCRIPTION
**Pull request by AmyJenSandoval**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,38 +1,41 @@
...
+        "react": "^18.2.0",
+        "react-dom": "^18.2.0",
+        "react-scripts": "5.0.1",
+        "web-vitals": "^2.1.4"
+    },
+    "scripts": {
+        "start": "react-scripts start",
+        "build": "react-scripts build",
+        "test": "react-scripts test",
+        "eject": "react-scripts eject"
```
Comment (Leigh-K) [inline package.json]: Lets add a new script here so people can run prettier from the CLI, we can also use it in the future for when we want to use prettier as a pre commit git hook
```
"prettier": "prettier --write ./src"
```
Then we can run `npm run prettier` which will fix all files

**Review by Leigh-K — COMMENTED** [review]
Beautiful stuff. Now I never need to see a semi colon again!.
If you could just add the script i mentioned and the cursor stuff to the buttons ill then merge this!

File: package.json (hunk 1 above)
Comment (AmyJenSandoval) [reply package.json]: Done

File: src/modules/registration/styles/Registration.module.css [hunk 2]
```diff
@@ -26,12 +26,16 @@
     border-radius: 3.125em;
     font-size: x-large;
     color: #fff;
+    cursor: pointer;
 }
 
 .returnToLoginButton {
     margin-bottom: 4em;
+    width: fit-content;
```
Comment (Leigh-K) [inline src/modules/registration/styles/Registration.module.css:34]: Interesting, i never use this and there is probably a better way to do what you are trying to do, but ima merge this anyway and DM you later when i know what it is

**Review by Leigh-K — APPROVED** [review]

File: src/modules/registration/styles/Registration.module.css (hunk 2 above)
Comment (AmyJenSandoval) [reply src/modules/registration/styles/Registration.module.css:34]: The button was the width of the screen but had no background and no border, so I wanted the button to be the width of the text only.

**Pull request merged**